### PR TITLE
make schemename configurable

### LINKED ISSUE #22: Add configuration for URI scheme
As there's no sane way to know the client request's URI scheme (`http` or `https`), we need a respective configuration.

### PR DESCRIPTION
**Pull request by robertkowalski**
support http per default and also https

closes #22

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ module.exports = function() {
   var defaultConfig = {
     host: '127.0.0.1',
     port: 1337,
+    schemename: 'http',
```
Comment (KlausTrainer) [inline lib/config.js]: This should either be in camel case (i.e., `schemeName`), or just be named `scheme`.  Although I tend to prefer the latter, I have no big opinion on which one we go for.

File: test/https.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        email: expectedToEmail,
+        username: 'Local Foo Bator'
+      }
+    }, function(err, response, body) {
+      t.equal(response.statusCode, 200);
+      t.ok(body.ok);
+      t.equal(actualFromEmail, expectedFromEmail);
+      t.equal(actualToEmail, expectedToEmail);
+      t.ok(emailBody.indexOf('Hi Local Foo Bator') !== -1, 'mailbody');
+      t.ok(emailBody.indexOf('https:\/\/') !== -1, 'https protocol is used');
```
Comment (KlausTrainer) [inline test/https.js]: s/protocol/scheme/

Comment (robertkowalski) [inline test/https.js]: fixed!

**Comment by KlausTrainer** [comment]
Thanks :)

**Pull request merged**